packages/auth-frontend-scripts/src/__tests__/build.test.ts

```diff
@@ -44,29 +44,29 @@ describe('build', () => {
     mockReadFileSync.mockReturnValue(`
       <!doctype html>
         <html lang="en">
-          <body>
-            <script src="/static/js/2.eb5d7501.chunk.js">
-            </script><script src="/static/js/main.f9ee5131.chunk.js"></script>
-          </body>
+        <head>
+          <script
+            defer="defer"
+            src="https://dev.hub.asap.science/.auth/static/js/main.80665dab.js"
```

Comment:
Our Regex failed because .chunk. is no longer included in chunk files. I also don't know why we have gone from 2 chunks to 1